ui/src/app/pages/access/controlled-tier-card.tsx

```diff
@@ -170,12 +172,12 @@ export const ControlledTierCard = (props: {
       </FlexColumn>
       <FlexColumn style={styles.modulesContainer}>
         <ControlledTierStep
-          data-test-id='controlled-signed'
```

Comment:
Enzyme was keying on these component data-test-id's, but they were not being passed to the DOM, so RTL can not read them. Changed it to description and passed it to an alt tag for accessibility and for test selectors.